extensions/sql-assessment/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "sql-assessment",
+  "displayName": "%displayName%",
+  "description": "%description%",
+  "version": "0.1.0",
+  "publisher": "Microsoft",
+  "preview": true,
+  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/LICENSE.txt",
+  "icon": "images/sqlserver.png",
```

Comment:
I'd suggest getting an actual icon for your extension.

At the very least please use the new default image we have for extensions e.g. https://github.com/microsoft/azuredatastudio/blob/master/extensions/big-data-cluster/images/extension.png